web/src/__tests__/ingestion.servertest.ts

```diff
@@ -817,57 +817,6 @@ describe("/api/public/ingestion API Endpoint", () => {
     expect(dbTrace.length).toBe(1);
   });
 
-  it("should fail for auth errors", async () => {
```

Comment:
auth error here was produced by adding a score to an unknown trace. this does not return an error anymore. there are other tests that test where ingestion can manage partial auth errors within a batch, that's why I removed this test

Reply:
perfect